buildengine/build_kotlin.go

```diff
@@ -158,36 +158,7 @@ var scaffoldFuncs = scaffolder.FuncMap{
 	"imports": func(m *schema.Module) []string {
 		imports := sets.NewSet[string]()
 		_ = schema.Visit(m, func(n schema.Node, next func() error) error {
-			switch n := n.(type) {
```

Comment:
kotlin generated modules will use on-demand imports instead of file-level imports; this way we don't need to handle aliasing imports to disambiguate imported data types with the same name which cause compilation errors (e.g. `ftl.foo.Request` and `ftl.bar.Request`).